src/renderer/components/Config/PrereqsView.tsx

```diff
@@ -91,6 +91,44 @@ const PrereqsView = ({ sx }: Props) => {
           .
         </Typography>
       )
+    } else if (status.id === 'ps1ExecutionPolicy') {
+      status.description = (
+        <Typography fontSize={14}>
+          <span style={{ fontSize: 14, opacity: 0.6 }}>
+            {
+              // Check if the user agent contains PowerShell version information
+              window.navigator.userAgent.includes('PowerShell')
+                ? `Run the following command in PowerShell ${window.navigator.userAgent}:`
```

Comment:
I dont see the version, This message looks strange and repetitive. `Run the following command in PowerShell:`. As the instructions below are guide not directly commands.

![image](https://github.com/EtherealEngine/etherealengine-control-center/assets/10975502/07145790-171d-472a-a753-1e018e898db4)